modules/nf-core/taxonkit/name2taxid/meta.yml

```diff
@@ -0,0 +1,53 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "taxonkit_name2taxid"
+description: Convert taxon names to TaxIds
+keywords:
+  - taxonomy
+  - taxIds
+  - taxon name
+  - conversion
+tools:
+  - "taxonkit":
+      description: "A Cross-platform and Efficient NCBI Taxonomy Toolkit"
+      homepage: "https://bioinf.shenwei.me/taxonkit/"
+      documentation: "https://bioinf.shenwei.me/taxonkit/usage/#name2taxid"
+      tool_dev_url: "https://github.com/shenwei356/taxonkit"
+      doi: "10.1016/j.jgg.2021.03.006"
+      licence: ["MIT"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - name:
+      type: string
+      description: Taxon name to look up (provide either this or names.txt, not both)
+  - names_txt:
+      type: file
+      description: File with taxon names to look up, each on their own line (provide either this or name, not both)
+  - taxdb:
+      type: file
+      description: Taxonomy database unpacked from ftp://ftp.ncbi.nih.gov/pub/taxonomy/taxdump.tar.gz
```

Comment:
Maybe specify it should be the directory that get's supplied (presumably? from my knowledge of `taxdump.tar.gz`)

Reply:
In my case it's specifically that file that I use outside of this module.